src/components/FeeExplainer/FeeSliders.tsx

```diff
@@ -169,7 +169,7 @@ export const FeeSliders: React.FC<FeeSlidersProps> = ({
               customInput={Input}
               isNumericString={true}
               suffix={localeParts.postfix}
-              prefix={localeParts.prefix}
+              prefix='$'
```

Comment:
[question]: If the user currency symbol is also `$` but not USD (AUD, CAD) should we set it to `$USD` to avoid confusion?

```suggestion
              prefix={localeParts.prefix === '$' ? '$USD' : '$'}
```

Reply:
Ohh very smort indeed.
Think that absolutely doesn't hurt to disambiguate things for those non-USD dollar users.

Do you believe `$USD` to be more readable, or should we go with the conventional `USD` (no $) for the left side?

Reply:
hmm, my opinion is i have no opinion :D